cpp/ql/src/Likely Bugs/Underspecified Functions/ImplicitFunctionDeclaration.ql

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
At some point this .ql file will need a corresponding .qhelp adding.

Reply:
I will study the style guide related to .qhelp files and write it when I'm confident in the query's output.